zerver/lib/management.py

```diff
@@ -57,10 +57,10 @@ def add_user_list_args(self, parser, required=False, help=None, all_users_arg=Tr
                 help=all_users_help)
 
     def get_realm(self, options):
-        # type: (Dict[str, Any]) -> Optional[Realm]
+        # type: (Dict[str, Any]) -> Realm
         val = options["realm_id"]
         if val is None:
-            return None
+            raise CommandError("No realm specified. Aborting.")
```

Comment:
We don't want to make this change; the intended UX is that you don't need to pass a realm.